python/README.md

```diff
@@ -0,0 +1,141 @@
+# Chericat Python
+A set of python code to display Chericat capabilities related data on graphs
+
+Packages required and their tested versions to run the programs:
```

Comment:
Could you provide a list of commands a user must execute, possibly in a separate section here, to prepare the environment? This should include `pkg64`/`pkg64c` commands that install required packages.